pkg/util/util.go

```diff
@@ -29,3 +31,18 @@ func IsGPUVMI(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+// IsIpv6Disabled returns if IPv6 is disabled according sysctl
+func IsIpv6Disabled() bool {
+	ipv6Disabled, err := exec.Command("cat", "/proc/sys/net/ipv6/conf/default/disable_ipv6").Output()
```

Comment:
Perhaps, I'll be better if you'll open and read the file instead of calling `cat`
See an example.. here maybe: https://github.com/kubevirt/kubevirt/blob/master/pkg/virt-launcher/virtwrap/util/cpu_utils.go#L30

Reply:
Thanks Vladik, on it

Reply:
ah btw this is from other PR (its cherry-picked), i will try it on both

Reply:
Sweet man, it fixed Travis
Thanks a bunch